### PR TITLE
IO1-2949: fixed ui issues

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/components/applications/SSV/NewMyAccount/components/OperatorDashboard/OperatorDashboard.tsx [hunk 1]
```diff
@@ -75,7 +75,10 @@ const OperatorDashboard = () => {
     );
   });
 
-  const openSingleOperator = (listIndex: number) => {
+  const openSingleOperator = (listIndex: number, e: any) => {
+    if ( e.target === e.currentTarget.querySelector('.openExplorer') ){
```
Comment (guym-blox) [inline src/app/components/applications/SSV/NewMyAccount/components/OperatorDashboard/OperatorDashboard.tsx:79]: please remove direct access to dom and use Ref 
https://legacy.reactjs.org/docs/refs-and-the-dom.html

File: src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.styles.ts [hunk 2]
```diff
@@ -18,6 +18,8 @@ export const useStyles = makeStyles((theme: Theme) => ({
     marginBottom: 24,
   },
   BoxWrapper: {
+    display: 'flex',
+    alignItems: 'center',
```
Comment (guym-blox) [inline src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.styles.ts:22]: please adjust order from shortest to longest

File: src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx [hunk 3]
```diff
@@ -55,8 +55,8 @@ const SetOperatorFee = () => {
                 {/*  <LinkText text={'Provide metadata'} link={'https://docs.ssv.network/run-a-node/operator-node'} /> to increase your operator discoverability and*/}
                 {/*  chances of attracting other stakers.*/}
                 {/*</Grid>*/}
-                <Grid className={classes.BoxWrapper} xs={12}>
-                  <LinkText text={'Monitor your node'} link={'https://docs.ssv.network/run-a-node/operator-node'} /> to improve your operator performance and uptime.
+                <Grid className={classes.BoxWrapper} alignItems="center" xs={12}>
```
Comment (guym-blox) [inline src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx:58]: remove inline style

File: src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/components/OperatorDetails/OperatorDetails.tsx [hunk 4]
```diff
@@ -55,7 +55,7 @@ const OperatorDetails = (props: Props) => {
             </Grid>
         )}
         {!operator.is_deleted && !withoutExplorer && <Grid item className={classes.OperatorType}>
-          <ImageDiv onClick={openExplorer} image={'explorer'} width={20} height={20}/>
+          <ImageDiv onClick={openExplorer}  classIdentifier="openExplorer" image={'explorer'} width={20} height={20}/>
```
Comment (guym-blox) [inline src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/components/OperatorDetails/OperatorDetails.tsx:58]: remove direct access to dom

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx [hunk 5]
```diff
@@ -13,131 +13,140 @@ import ProcessStore from '~app/common/stores/applications/SsvWeb/Process.store';
 import NewWhiteWrapper from '~app/components/common/NewWhiteWrapper/NewWhiteWrapper';
 import NotificationsStore from '~app/common/stores/applications/SsvWeb/Notifications.store';
 import OperatorStore, { IOperator } from '~app/common/stores/applications/SsvWeb/Operator.store';
+import Tooltip from '@mui/material/Tooltip';
```
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx:16]: done

File: src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.styles.ts (hunk 2 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.styles.ts:22]: done

File: src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx (hunk 3 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx:58]: done

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx:16]: done

File: src/app/components/applications/SSV/NewMyAccount/components/ClusterDashboard/ClusterDashboard.tsx [hunk 6]
```diff
@@ -138,7 +139,9 @@ const ClusterDashboard = () => {
       </Grid>
       <Dashboard
           disable
-          rows={rows}
+          rows={rows.map((row: any) => {
+            return { ...row, clusterID: row.clusterID.slice(2) };
```
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/components/applications/SSV/NewMyAccount/components/ClusterDashboard/ClusterDashboard.tsx:143]: done

File: src/app/components/applications/SSV/NewMyAccount/components/Dashboard/Dashboard.tsx [hunk 7]
```diff
@@ -115,7 +115,7 @@ const Dashboard = (props: DashboardProps) => {
             {rows.map((row, index: number) => (
                 <CustomizedBodyRow
                     key={index}
-                    onClick={()=>{rowsAction && rowsAction(index);}}
+                    onClick={(e : any)=>{rowsAction && rowsAction(index, e);}}
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/NewMyAccount/components/Dashboard/Dashboard.tsx:118]: Remove redundant empty space after `e`

File: src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx [hunk 8]
```diff
@@ -16,80 +16,83 @@ import ApplicationStore from '~app/common/stores/applications/SsvWeb/Application
...
-                      {/*Enter an Ethereum address that will receive all of your validators block proposal rewards. <LinkText text={'What are proposal rewards?'} link={'http://google.com'} />*/}
+    return (
+        <BorderScreen
+            blackHeader
+            header={'Change Operator Name'}
+            body={[
+                (
+                    <Grid container item>
+                        <Grid container style={{ gap: 24 }}>
+                            <Grid item className={classes.Text}>
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx:70]: Keep one variant of code styling. After `{{` and before `}}` empty spaces are absent in most of the cases, keep it the same

File: src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx [hunk 9]
```diff
@@ -16,80 +16,83 @@ import ApplicationStore from '~app/common/stores/applications/SsvWeb/Application
...
+                                <InputLabel title="Operator Name"/>
+                                <TextInput
+                                    value={userInput}
+                                    disable={readOnlyState}
+                                    showError={!isAddressValid}
+                                    data-testid="new-operator-name"
+                                    onChangeCallback={setOperatorName}
+                                    icon={<Grid onClick={() => setReadOnlyState(false)} className={classes.EditIcon}/>}
+                                />
+                                {/*<Grid className={classes.ErrorText}>{!isAddressValid ? 'Invalid address, please input a valid Ethereum wallet address' : ''}</Grid>*/}
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx:85]: If this should not be used just remove it

File: src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx [hunk 10]
```diff
@@ -56,7 +56,7 @@ const SetOperatorFee = () => {
                 {/*  chances of attracting other stakers.*/}
                 {/*</Grid>*/}
                 <Grid className={classes.BoxWrapper} xs={12}>
-                  <LinkText text={'Monitor your node'} link={'https://docs.ssv.network/run-a-node/operator-node'} /> to improve your operator performance and uptime.
+                  <LinkText text={'Monitor your node'} link={'https://docs.ssv.network/run-a-node/operator-node/maintenance/monitoring-grafana'} /> to improve your operator performance and uptime.
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx:59]: Move this link to constants

File: src/app/components/applications/SSV/RegisterValidatorHome/components/FundingPeriod/FundingPeriod.tsx [hunk 11]
```diff
@@ -30,7 +30,7 @@ const FundingPeriod = () => {
   const ssvStore: SsvStore = stores.SSV;
   const processStore: ProcessStore = stores.Process;
   const operatorStore: OperatorStore = stores.Operator;
-  const [customPeriod, setCustomPeriod] = useState(1);
+  const [customPeriod, setCustomPeriod] = useState(730);
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/RegisterValidatorHome/components/FundingPeriod/FundingPeriod.tsx:33]: Move custom period value to constants (when I say constants i mean config.ts file)

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx [hunk 12]
```diff
@@ -14,130 +15,138 @@ import NewWhiteWrapper from '~app/components/common/NewWhiteWrapper/NewWhiteWrap
...
+        setSelectedBox(id);
+    };
 
-  const checkBox = (id: number) => {
-    setTextCopied(false);
-    setSelectedBox(id);
-  };
+    const copyButton = () => {
+        if (!textCopied) return <Grid item className={classes.CopyButton} onClick={copyToClipboard}>Copy</Grid>;
+        return <Grid onClick={copyToClipboard} container item className={classes.ButtonCopied}>
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx:78]: Extract copy button to standalone component with properties passing to it

**Review by meshin-blox — COMMENTED** [review]
@axelrod-blox go over my comments and fix them in additional pr

**Comment by meshin-blox** [comment]
@guym-blox @axelrod-blox when merging branches delete them right away

File: src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx (hunk 10 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/OperatorSuccessPage/OperatorSuccessPage.tsx:59]: done

File: src/app/components/applications/SSV/RegisterValidatorHome/components/FundingPeriod/FundingPeriod.tsx (hunk 11 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/FundingPeriod/FundingPeriod.tsx:33]: done

File: src/app/components/applications/SSV/NewMyAccount/components/OperatorDashboard/OperatorDashboard.tsx (hunk 1 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/NewMyAccount/components/OperatorDashboard/OperatorDashboard.tsx:79]: done

File: src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/components/OperatorDetails/OperatorDetails.tsx (hunk 4 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/components/OperatorDetails/OperatorDetails.tsx:58]: done

File: src/app/components/applications/SSV/NewMyAccount/components/Dashboard/Dashboard.tsx (hunk 7 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/NewMyAccount/components/Dashboard/Dashboard.tsx:118]: done

File: src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx (hunk 8 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx:70]: done

File: src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx [hunk 13]
```diff
@@ -16,80 +16,83 @@ import ApplicationStore from '~app/common/stores/applications/SsvWeb/Application
...
-                <Grid container item>
-                  <Grid container style={{ gap: 24 }}>
-                    <Grid item className={classes.Text}>
-                      {/*Enter an Ethereum address that will receive all of your validators block proposal rewards. <LinkText text={'What are proposal rewards?'} link={'http://google.com'} />*/}
+    return (
+        <BorderScreen
+            blackHeader
+            header={'Change Operator Name'}
+            body={[
+                (
```
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx:67]: component BorderScreen need array for body parameter, if we do not send array, we get warning

File: src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx (hunk 9 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/NewMyAccount/components/changeOperatorName/ChangeOperatorName.tsx:85]: done

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx [hunk 14]
```diff
@@ -14,130 +15,138 @@ import NewWhiteWrapper from '~app/components/common/NewWhiteWrapper/NewWhiteWrap
...
+    const isSelected = (id: number) => selectedBox === id;
+    const goToNextPage = () => navigate(config.routes.SSV.VALIDATOR.DISTRIBUTION_METHOD.UPLOAD_KEYSHARES);
+    const {
+        operatorsIds,
+        operatorsKeys,
+    } = Object.values(operatorStore.selectedOperators).sort((a: any, b: any) => a.id - b.id).reduce((aggr, operator: IOperator) => {
+        // @ts-ignore
+        aggr.operatorsIds.push(operator.id);
+        // @ts-ignore
+        aggr.operatorsKeys.push(operator.public_key);
```
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx:38]: done

File: src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx (hunk 12 above)
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx:78]: done

**Pull request merged**